codespell_lib/data/dictionary.txt

```diff
@@ -12074,6 +12074,7 @@ examinated->examined
 examing->examining
 examinining->examining
 examles->examples
+examlpe->example
```

Comment:
If you don't mind adding the plural `examlpes->examples`, see https://github.com/search?q=examlpes&type=code